raiden/network/pathfinding.py

```diff
@@ -183,6 +185,28 @@ def get_pfs_iou(
         raise ServiceRequestFailed(str(e))
 
 
+def request_last_iou(
+        url: str,
+        token_network_address: typing.Union[typing.TokenNetworkAddress, typing.TokenNetworkID],
+        sender: typing.Address,
+        receiver: typing.Address,
+) -> typing.Optional[typing.Dict]:
+
+    timestamp = datetime.now().isoformat(timespec='seconds')
+    signature = eth_sign_hash_message(
+        Web3.toBytes(hexstr=sender) + receiver + bytes(timestamp, 'utf-8'),
+    )
+
+    return get_pfs_iou(
```

Comment:
`request_last_iou` and `get_pfs_iou` are so small and tightly coupled that I would merge them into a single function. But that might just be my personal preference.